apps/api/src/app/markers/dto/create-marker.input.ts

```diff
@@ -0,0 +1,19 @@
+import { InputType, Int, Field } from '@nestjs/graphql';
+import { IsEnum, IsNotEmpty } from 'class-validator';
+import { MarkerType } from '../entities/marker.entity';
+
+@InputType()
+export class CreateMarkerInput {
+  @Field(() => MarkerType, { description: 'Marker Type' })
+  @IsEnum(MarkerType)
+  @IsNotEmpty()
+  type: MarkerType;
+
+  @Field({ description: 'Marker Name' })
+  @IsNotEmpty()
+  name: string;
+
+  @Field({ description: 'Marker Name' })
+  @IsNotEmpty()
```

Comment:
Should that be mandatory? Or is that only additional information besides a default color?

Reply:
Added default value